contrib/signet/README.md

```diff
@@ -17,3 +17,64 @@ Syntax: `getcoins.py [-h|--help] [-c|--cmd=<bitcoin-cli path>] [-f|--faucet=<fau
 If using the default network, invoking the script with no arguments should be sufficient under normal
 circumstances, but if multiple people are behind the same IP address, the faucet will by default only
 accept one claim per day. See `--password` above.
+
+miner
+=====
+
+To mine the first block in your custom chain, you can run:
+
+  cd src/
+  CLI="./bitcoin-cli -conf=mysignet.conf"
+  MINER="..contrib/signet/miner"
+  GRIND="./bitcoin-util grind"
+  ADDR=$($CLI -signet getnewaddress)
+  $MINER --cli="$CLI" generate --grind-cmd="$GRIND" --address="$ADDR" --set-block-time=-1
+
+This will mine a block with the current timestamp. If you want to backdate the chain, you can give a different timestamp to --set-block-time.
+
+You will then need to pick a difficulty target. Since signet chains are primarily protected by a signature rather than proof of work, there is no need to spend as much energy as possible mining, however you may wish to choose to spend more time than the absolute minimum. The calibrate subcommand can be used to pick a target, eg:
+
+  $MINER calibrate --grind-cmd="$GRIND"
+  nbits=1e00f403 for 25s average mining time
```

Comment:
nbits=1e00f403 > --nbits=1e00f403

Reply:
That line is an example of output of the command

Reply:
The formatting looks strange when reading the README.md

Reply:
Ah, apparently 4 spaces is the standard, not wordpress's 2 spaces. Fixed.